README.md

```diff
@@ -4,30 +4,8 @@ Credit to SeanDaBlack for the basis of the script.
 req.py is selenium python bot.
 sc.js is a the base of the ios shortcut [COMING SOON]
 
-# Setup
-
-On mac/pc:
+# Setup (Mac/Windows/Linux)

```

Comment:
This is great that you've been able to streamline the process for chromedriver as such. Code related to this from this PR should be merged to main immediately.

Reply:
I created a seperate pull request due to this one becoming a bit buggy due to many different purposes.